src/App.tsx

```diff
@@ -3,33 +3,51 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
 
   render() {
+    const visibleMovies = moviesFromServer.filter((movie) => {
+      if (movie.title.toLowerCase().includes(this.state.query.toLowerCase())
+        || movie.description.toLowerCase().includes(this.state.query.toLowerCase())) {
+        return true;
+      }
+
+      return false;
+    });
```

Comment:
You can simplify this
```suggestion
    const visibleMovies = [...moviesFromServer].filter(({ title, description })  => {
      const lowerCaseQuery = query.toLowerCase();
      
      return title.toLowerCase().includes(lowerCaseQuery)
        || description.toLowerCase().includes(lowerCaseQuery);
    });
```

Reply:
Create a separate function for this logic, as well.